addons/bestja_detailed_reports/views/detailed_reports.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record model="ir.ui.view" id="bestja_detailed_report">
+            <field name="name">bestja.detailed_report.form</field>
+            <field name="model">bestja.detailed_report</field>
+            <field name="priority" eval="0"/>
+            <field name="arch" type="xml">
+                <form>
+                    <header>
+                        <field name="id" invisible="1"/>
+                        <field name="user_can_moderate" invisible="1"/>
+                        <div attrs="{'invisible': [('id', '=', False)]}">
```

Comment:
You've got three `<div attrs="{'invisible': [('id', '=', False)]}">`. It would be really easy to group them together and merge into one.